backend/main.py

```diff
@@ -31,13 +31,13 @@ def read_match_genesets(pattern: str, db: Session = Depends(get_db)):
 
 @app.get("/genesets/{geneset_id}", response_model=schemas.Geneset)
 def read_geneset(geneset_id: int, db: Session = Depends(get_db)):
-
     return crud.get_geneset(db, geneset_id)
 
-@app.put("/genesets/{geneset_id}")
-def update_genesets(geneset_id: int, db: Session = Depends(get_db)):
 
-    return crud.get_geneset(db, geneset_id)
+@app.put("/genesets/update/{geneset_id}", response_model=schemas.Geneset)
```

Comment:
its not a really good practice to have the name of the method in the url. do we really need to have /update ?

Reply:
Nope, indeed. I wanted to follow the `search` scheme but actually that was a bad idea, I'm editing this.